man/docker-build.1.md

```diff
@@ -66,6 +67,11 @@ set as the **URL**, the repository is cloned locally and then sent as the contex
 **--force-rm**=*true*|*false*
    Always remove intermediate containers, even after unsuccessful builds. The default is *false*.
 
+**--isolation**="*default*"
+   Isolation specifies the type of isolation technology used by containers. On Linux, the only supported option is 'default' (Linux Namespaces). Windows supports 'default' (the daemon default specified by `--exec-opt isolation=`), 'process' (namespace isolation only), or 'hyperv' (Hyper-V hypervisor partition based isolation).
```

Comment:
Rather than trying to cover all this in the  flag, please keep the command simple and add the content you created for `build` above here in the man page.